x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/quick_start_tab.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiSteps } from '@elastic/eui';
+
+import { useQuickStartCreateForm, useWaitForFleetServer } from './hooks';
+
+import {
+  getGettingStartedStep,
+  getConfirmFleetServerConnectionStep,
+  getInstallFleetServerStep,
+} from './steps';
+
+export const QuickStartTab: React.FunctionComponent = () => {
+  const quickStartCreateForm = useQuickStartCreateForm();
+  const { isFleetServerReady } = useWaitForFleetServer();
+
+  const steps = [
+    getGettingStartedStep({
+      quickStartCreateForm,
+    }),
+    getInstallFleetServerStep({
+      isFleetServerReady,
+      disabled: quickStartCreateForm.status !== 'success',
```

Comment:
Quick note on disabled/complete status for these steps: since the designs call for showing/hiding individual steps based on current form state, we allow a `disabled` prop to be passed in to support this use case. If a given step needs to implement a "complete" state (only a subset of steps do this), it's done within the component itself.